packages/terra-markdown/src/Markdown.jsx

```diff
@@ -19,7 +19,7 @@ const defaultProps = {
 const Markdown = props => (
   <div
     dir="ltr"
-    className="markdown-body"
+    className={'markdown-body'}
```

Comment:
Yeah, in the future I plan to drop the dependency on github-markdown-css and write our markdown styles.